src/app/models/AnalysisJobItem.ts

```diff
@@ -49,9 +50,19 @@ export class AnalysisJobItem extends AbstractModel implements IAnalysisJobItem {
     super(analysisJobItem, injector);
   }
 
+  // most likely "/analysis_jobs/{analysisJobId}/audio_recordings/{analysisJobItemId}"
   public get viewUrl(): string {
-    // Potentially "/audio_analysis/{analysisJobId}/results{path}"?
-    throw new Error("AnalysisJobItem viewUrl not implemented.");
+    return audioAnalysisItemMenuItem.route.format({
+      analysisJobId: this.analysisJobId,
+      analysisJobItemId: this.id,
+    });
+  }
+
+  public get adminViewUrl(): string {
+    return audioAnalysisItemMenuItem.route.format({
+      analysisJobId: this.analysisJobId,
+      analysisJobItemId: this.id
+    });
```

Comment:
why define this function if we don't have a separate view for it?

Reply:
100% agree.

I mistakenly thought this required by AbstractModel